docs/content/cluster-scope/add_operator.md

```diff
@@ -0,0 +1,32 @@
+# Constructing a PR to add an Operator
+
+There are two methods one can follow to add an Operator to a cluster:
+1. Install via OLM
+1. Manual install
+
+## Install an operator via OLM
```

Comment:
Let's throw in a pre-requisite section, what should the person reading this need to know/have prepared before attempting this? 

Seems to me they should at least know: 

1. Basic git knowledge
2. Familiarity with [kustomize](https://kustomize.io/) 
3. Some basic OCP/K8S knowledge and OLM
4. What operator needs to be deployed, the version that's needed, the operator channel (maybe link to [this](https://olm.operatorframework.io/docs/concepts/)...I just found this, seems like a solid source for olm/operator concepts)